packages/engine-client/src/types/serverSubscriptionEventTypes.ts

```diff
@@ -3,7 +3,7 @@ import { EngineServerPriceTickLiquidity } from './serverQueryTypes';
 /**
  * @description Event from subscribing to a `trade` stream.
  */
-export interface SubscriptionTradeEvent {
+export interface EngineServerSubscriptionTradeEvent {
```

Comment:
I'd like to prefix these with `EngineServer` to standardize with query / executes

Reply:
yea, i think change looks good as well 👍